src/shims/unix/linux/epoll.rs

```diff
@@ -215,8 +214,11 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
             interest_list.insert(epoll_key, event);
 
             target_file_description.check_readiness(this).map(|events| {
-                this.update_readiness(events, target_file_description.return_epoll_events()?)
-            })??;
+                target_file_description.update_readiness(
```

Comment:
This won't work because ``update_readiness`` is only implement for ``SocketPair`` and ``Event`` but not general file description.